File: 23_C#Basics/_03_ComplexConditionalStatements/_03_AnimalType/_03_AnimalType.cs
using System;

public class _03_AnimalType
{
    public static void Main(string[] args)
    {
        // Read animal input value
        string animal = Console.ReadLine().ToLower();

        // Determine whether the animal is mammal, reptile or unknown
        switch (animal)
        {
            case "dog":
                Console.WriteLine("mammal");
                break;
            case "crocodile":
            case "tortoise":
            case "snake":
                Console.WriteLine("reptile");
                break;
            default:
                Console.WriteLine("unknown");
                break;
        }
    }
}

File: 01_ProgrammingBasics/_03_ComplexConditionalStatements/_03_AnimalType.java
package _01_ProgrammingBasics._03_ComplexConditionalStatements;

import java.util.Scanner;

public class _03_AnimalType {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read animal input value
        String animal = scan.nextLine().toLowerCase();

        // Determine whether the animal is mammal, reptile or unknown
        switch (animal) {
            case "dog":
                System.out.println("mammal");
                break;
            case "crocodile":
            case "tortoise":
            case "snake":
                System.out.println("reptile");
                break;
            default:
                System.out.println("unknown");
                break;
        }

        scan.close();
    }
}
